updates/testing.json

```diff
@@ -12,21 +12,11 @@
         }
       }
     },
-    {
-      "version": "30.20191014.0",
-      "metadata": {
-        "rollout": {
-          "start_percentage": 100
```

Comment:
Dropping the rollout section entirely is on purpose, right? And to confirm, nodes that check in now would still get it even without it?

Reply:
1. Yes it's on purpose. Both rollouts are at 100% already, so the oldest one can be safely dropped. The newest one stays, it is just converted to a barrier.
 2. Yes, barriers are active as long as they exist. They don't need an explicit 100% rollout in place to be active.